web/src/lib/components/BidList.svelte

```diff
@@ -6,7 +6,34 @@
     export let auction: Auction;
 </script>
 
-<div class="overflow-x-auto w-full">
+<div class="md:hidden w-full rounded-b-lg mb-2">
+    <div class="w-full bg-black/10 rounded-t-lg py-2 mb-1">
+        <span class="ml-5 text-xl">Bids</span>
+    </div>
+        <bids class="pt-2">
```

Comment:
what is `<bids` ? :)

Reply:
was supposed to say `<div` there, just a typo... didn't seem to influence how the thing rendered, though! Corrected it now.

Reply:
Yeah, it seems like Tailwind treats all elements equally pretty much, but it's better to stick to as semantic as possible still.